lib/dx/error/timeout.ex

```diff
@@ -1,7 +1,13 @@
 defmodule Dx.Error.Timeout do
-  defexception []
+  defexception [:configured_timeout]
 
-  def message(_error) do
-    "A timeout occurred while loading the data required."
+  def message(error) do
+    """
+    A timeout occurred while loading the data required.
+    The timeout is currently set to #{error.configured_timeout}.
+    If you are using the default `Dx.Loaders.Dataloader`, you can
+    increase the timeout by passing `loader_options: [timeout: your_timeout]`
+    to the `Dx.get/3` or whichever entry function you are using.
```

Comment:
`Dx.get` is not a good example, because it doesn't load any data 😆 

```suggestion
    as last argument to the `Dx` function you are using, such as `Dx.load/3`.
```